package.json

```diff
@@ -113,6 +113,7 @@
     "lodash": "^4.17.14",
     "mathjs": "^6.6.4",
     "moment": "^2.24.0",
-    "wavesurfer.js": "3.3.3"
+    "wavesurfer.js": "3.3.3",
+    "nerdamer": "1.1.4"
```

Comment:
Sort alphabetically.

Reply:
Also, please run `yarn` to update `yarn.lock`. For context, we no longer update the `yarn.lock` file automatically so that it doesn't affect dev experience.

Reply:
Done both.